src/routes/search.js

```diff
@@ -136,9 +168,7 @@ class SearchRoute extends Component {
               onSearch={this.handleSearch}
             />
           )}
-        >
-          {this.props.children}
```

Comment:
Why not use `SearchPaneset`'s children to pass embedded components instead of the `resultsView` attribute. Actually I think I see. Is it because there are two "slots" results and detail?

Incidentally, this is exactly what named blocks are for in Ember https://github.com/machty/rfcs/blob/named-blocks/text/0000-named-blocks.md

Reply:
Yes, it is because there are two "slots" to render. The details and results are side-by-side in the UI and the logic for showing either or both is inside the `SearchPaneset` component